metrics.go

```diff
@@ -182,6 +182,21 @@ const (
 	// MetricBackendWriteFailedRequests measures failed backend write requests count
 	MetricBackendWriteFailedRequests = "backend_write_requests_failed_total"
 
+	// MetricBackendAtomicWriteRequests measures backend atomic write requests count
+	MetricBackendAtomicWriteRequests = "backend_atomic_write_requests_total"
```

Comment:
Do you think it would be worth to also track the number of actions per atomic write request?